contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,177 @@
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What Is Angular?
+
+Angular is one of the popular Javascript frameworks used for building web applications.
+The framework aims to make development and testing of web applications easier. Angular is maintained by Google, as well as a community of developers.
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+<sub>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a></sub>
+
+Angular is a complex framework and can be challenging for new developers to understand.
+However, given below are some noteworthy features of Angular (adapted from [here](https://hackr.io/blog/why-should-you-learn-angular-in-2018)):
+
+1. **Angular supports Cross Platform Development.** Apart from web applications, the more recent versions of Angular can be used to build <tooltip content="a smartphone application that is coded in a specific programming language, such as Swift for iOS or Java for Android operating systems">native mobile applications</tooltip> and Desktop applications.
+ With Angular 5 and above, progressive web applications can also be developed. These applications have high performance and can work offline as well.
+
+2. **Angular uses TypeScript.** Angular applications are built using the TypeScript language, a superset of JavaScript. TypeScript provides <tooltip content="Static typing involves type checking during compile-time as opposed to run-time. Types are associated with variables not values.">static typing</tooltip>. This helps the compiler show warnings about any potential errors in the code, before the app runs. Another advantage is that TypeScript provides code completion using <tooltip content="IntelliSense provides active hints as code is added">IntelliSense.</tooltip>
+
+    Below is an example of static typing using TypeScript.
+    
+    ```typescript
+    // TypeScript
+    let name: string; // <-- static typed!
+    ```
+    
+    ```javascript
+    // JavaScript (ES6)
+    let name;
+    ```
+    <box type="info"> TypeScript was created and is maintained by Microsoft. It has been <a href="https://developer.telerik.com/featured/the-rise-of-typescript/">growing in popularity</a> over the last few years making it reliable to use.</box>
+                                                  
+3. **Angular is backed by Google.** Google announced Long-Term Support (LTS) for Angular back in 2017. This means Google plans to further scale up the Angular ecosystem.
```

Comment:
This is what [Angular website](https://angular.io/docs) says:

>Angular is a platform that makes it easy to build applications with the web. Angular combines declarative templates, dependency injection, end to end tooling, and integrated best practices to solve development challenges. Angular empowers developers to build applications that live on the web, mobile, or the desktop

An alternative you can consider:, Start the WHAT section by simply quoting that statement (it's from the creators of Angular after all) and then go onto explain those characteristics mentioned.
* declarative templates (you'll probably need to explain the helloworld for this one)
* dependency injection
* end to end tooling
* integrated best practices 
* applications that live on the web, mobile, or the desktop

In fact, the first two sound like characteristics while the other three can be given as benefits (i.e., in the WHY section). You can also say 'others have proposed many other benefits' and give some links to such articles. I don't think you have enough space to explain all of them in detail.

If you follow the above, your article has a solid and authoritative basis. The current article doesn't match very well with the Angular definition given above.